arm-appinsights/2015-05-01/swagger/syntheticMonitor_API.json

```diff
@@ -0,0 +1,402 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AppInsightsManagementClient",
+    "description": "Azure Application Insights client for synthetic monitor based alerting.",
+    "version": "2015-05-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.insights/webtests": {
+      "get": {
+        "description": "Get all Application Insights synthetic monitors defined within a specified resource group.",
+        "operationId": "SyntheticMonitor_ListByResourceGroup",
```

Comment:
- [x] **Done** I've completely removed the use of the term 'Synthetic Monitor' with my upcoming change. Synthetic monitor is a term we use throughout our documentation, but WebTests is the name of the resource. I believe my teammates have some thoughts on this subject and if they can come up with a compelling reason for me to change it back I will do so.